### DISCUSSION TITLE
Getting error on callback

### DISCUSSION THREAD

**Discussion by casoetan** [lrstanley/chix#34] [Q&A]
I'm getting the following error on provider/auth callback

`Internal Server Error: could not find a matching session for this request` Looks like that's happening on this line 

guser, err := gothic.CompleteUserAuth(w, gothic.GetContextWithProvider(r, chi.URLParam(r, "provider")))

I guess I'm missing some configuration for sessions/cookies

**Comment by lrstanley** [comment]
FWIW, if anyone else happens to stumble upon this, an issue was created and discussion is happening there: https://github.com/lrstanley/chix/issues/35
